code/modules/food_and_drinks/kitchen_machinery/processor.dm

```diff
@@ -58,6 +58,8 @@
 		var/cached_multiplier = (recipe.food_multiplier * rating_amount)
 		for(var/i in 1 to cached_multiplier)
 			var/atom/processed_food = new recipe.output(drop_location())
+			var/atom/input_food = new recipe.input
+			input_food.reagents.copy_to(processed_food, input_food.reagents.total_volume)
```

Comment:
What happens to `input_food` after the proc is done?
Does it stay in nullspace? You may need to clean it up afterwards

Reply:
`input_food` has been replaced with `what`

`what` is already cleaned at the end of this proc

all is well